ci/long_running_tests/workloads/many_actor_tasks.py

```diff
@@ -45,8 +45,10 @@ def method(self):
         self.value += 1
 
 
+# TODO (williamma12): Update the actors to each have only 0.1 of a cpu once
+# https://github.com/ray-project/ray/issues/4312 gets resolved.
 actors = [
-    Actor._remote([], {}, num_cpus=0.1, resources={str(i % num_nodes): 0.1})
+    Actor._remote([], {}, num_cpus=1, resources={str(i % num_nodes): 0.1})
```

Comment:
Unfortunately I tried running it and it still doesn't run (I just tried it out).

Let's revert all the changes and just change the remote decorator to have `num_cpus=0.1` and remove the `num_cpus` from the `_remote`

Reply:
Sounds good! I did that and it works as expected now